utils/keyprovider/keprovider.pb.go

```diff
@@ -0,0 +1,243 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: proto.proto
+
+package keyproviderpb
```

Comment:
directory should match pacakge name, so maybe change the folder to keyproviderpb?

Reply:
done